.github/FEATURE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+**Description of Problem**:
+Short overview of the current situation. 
+Why is this feature needed?
```

Comment:
maybe suggest to link to any relevant forum threads

Reply:
I would put them under the section `example`. We could add the suggestion over there. Maybe we should rename that section to `resources` or something like that to make it more clear.